website/article/convolution/index.html

```diff
@@ -6,10 +6,17 @@
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Fluid: Data-Linked Visualisations</title>
+<<<<<<< HEAD
```

Comment:
It looks like something got accidentally committed before the merge conflict was resolved. You need to delete either the "Current change" or the "Incoming change". I think you want the versions with the `"../" prefixes on the URLs.